README.md

```diff
@@ -1,54 +1,239 @@
+<br>
+
+*Here is our filter function: it's a standard Python function that will constitute our unique task in the scenario*
+```python
+def filtering_genre(initial_dataset: pd.DataFrame, selected_genre):
+    filtered_dataset = initial_dataset[initial_dataset['genres'].str.contains(selected_genre)]
+    filtered_data = filtered_dataset.nlargest(7, 'Popularity %')
+    return filtered_data
+```
+
+*This is the execution graph of the scenario we will be implementing*
+
+<div align="center"><img src="readme_img/readme_exec_g.png" alt="Taipy Logo"  width="400" height="300"/></div> 
+
+
+### Taipy Studio - The easy peasy way
+*You can use the Taipy Studio extension in VSCode to configure your pipeline easily* 
+
+<div align="center"><img src="readme_img/readme_studio.gif" width=600 height=400 alt="GUI demo"></img></div> 
+
+*Your configuration is automatically saved as a toml file* 
+
+<br>
+<br>
+
+### <div align="center">*Want to be Studio-us?*</div>
+*<div align="center">Check out our [Getting Started](https://docs.taipy.io/en/latest/getting_started/getting-started-core/) and [Documentation](https://docs.taipy.io/en/latest/manuals/studio/) </div>*
+
+<br>
+<br>
+<br>
+<br>
 
-## Contributing
+### Taipy CORE - a walk on the code side
+<div align="left">If you prefer coding your configurations instead of using Taipy Studio, Taipy's got you 🫵 </div>   
+
+*<div align="left">Check out the movie genre demo scenario creation with this [Demo](https://www.taipy.io/project/movie-genre-selector/) </div>*
+
+<br>
+<br>
+<br>
+
+### <div align="center">*Gimme Core*</div>
+*<div align="center">Check out our [Getting Started](https://docs.taipy.io/en/latest/getting_started/getting-started-core/) and [Documentation](https://docs.taipy.io/en/latest/manuals/core/) </div>*
+
+<br>
+<br>
+<br>
+
+
+## Front-end ➕ Back-end
+*Now, let's load this configuration and add a GUI on top for a 🎉FULL APPLICATION🎉*
+```python
+import taipy as tp
+import pandas as pd
+from taipy import Config, Scope, Gui
+
+# TAIPY Core
+
+# Filtering function - task
+def filtering_genre(initial_dataset: pd.DataFrame, selected_genre):
+    filtered_dataset = initial_dataset[initial_dataset['genres'].str.contains(selected_genre)]
+    filtered_data = filtered_dataset.nlargest(7, 'Popularity %')
+    return filtered_data
+
+# Load the configuration made with Taipy Studio
+Config.load('config.toml')
+scenario_cfg = Config.scenarios['scenario']
+
+# Start Taipy Core service
+tp.Core().run()
+
+# Create a scenario
+scenario = tp.create_scenario(scenario_cfg)
+
+
+# TAIPY GUI
+# Let's add Taipy GUI to our Taipy Core for a full application
+
+# Callback definition - submits scenario with genre selection
+def modify_genre(state):
```

Comment:
modify_genre -> on_genre_selected

Reply:
Callbacks are one of the exceptions to the rule in my previous comment... The rule is to prefix by `on` the nominal form of the action that triggers the execution of the callback.
`on_action`, `on_genre_selected`, `on_click`, ...